### DISCUSSION TITLE
Translating the copyright generator

### DISCUSSION THREAD

**Discussion by bigsavv** [squidfunk/mkdocs-material#5770] [Ideas]
Hey! I thought it would be nice to have the generator that takes care of the _Made with_ [_Material for MkDocs_](https://squidfunk.github.io/mkdocs-material/) copyright to be translated at the same time as the rest of the docs.

I think many would love to keep the message on their docs but might reconsider if it's in English.

Thanks for this very good tool !!

**Comment by squidfunk** [comment]
Thanks for suggesting! it's currently not planned and you're the first one that suggested it. Maybe we'll reconsider if a significant amount of users wants that 😊

**Comment by bigsavv** [comment]
Thank you for the answer ! For the time being, I'll try and modify it through the [customization options](https://squidfunk.github.io/mkdocs-material/customization/?h=cust#overriding-blocks) by modifying a block.

Cheers !!
